examples/annotated-http-service-kotlin/src/main/kotlin/example/armeria/server/annotated/kotlin/ContextAwareService.kt

```diff
@@ -18,18 +18,13 @@ class ContextAwareService {
         // Make sure that current thread is request context aware
         ServiceRequestContext.current()
 
-        // Propagate armeria request context.
-        // `BusinessLogic.blockingTask()` is designed to be dispatched to `BusinessLogic.myDispatcher`
-        // to limit the number of parallel executions.
-        // Thus, we have to pass a request context as `ThreadContextElement`.
-        withContext(ArmeriaRequestContext()) {
-            log.info("Start blocking task for $name")
-            BusinessLogic.blockingTask()
-            log.info("Finished blocking task for $name")
+        // Armeria request context will automatically be propagated to myDispatcher's scope.
+        log.info("Start blocking task for $name")
+        BusinessLogic.blockingTask()
+        log.info("Finished blocking task for $name")
 
-            // Make sure that current thread is request context aware
-            ServiceRequestContext.current()
-        }
+        // Make sure that current thread is request context aware
+        ServiceRequestContext.current()
```

Comment:
Could remove these two lines.

Reply:
A thread could be changed after resuming a suspend function.
It would be better to leave it as it is.

Reply:
Oops... I didn't know that there is another `ServiceRequestContext.current()` below. 😱